website/data/internals-nav-data.json

```diff
@@ -1,4 +1,5 @@
 [
+  { "heading": "Terraform Internals" },
   {
```

Comment:
I don't think the content on `/internals` is important enough to warrant this, but it might still be a good idea to add an "Overview" node to match the CLI and Language sidebars.